.github/workflows/golang_lint_and_test.yml

```diff
@@ -4,14 +4,12 @@ on: [push, pull_request]
 jobs:
   golang_lint_and_test:
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
     steps:
       - uses: actions/checkout@v2
       - name: Codespell
         uses: codespell-project/actions-codespell@master
         with:
-          ignore_words_list: "actualy,nwe"
+          ignore_words_list: "actualy,nwe,caesar"
```

Comment:
Why are actualy and nwe ignored?

Reply:
The `actualy` is in fact `actualY` in the code🤷🏼‍♂️ the code spell has some issues like that.